packages/regenesis-surgery/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
You can avoid `cd ../..` by creating a subshell with `( )`, for example
```bash
(
  cd ...
)
# still in same directory
```

There is also `pushd` and `popd` for this